### DISCUSSION TITLE
Github Support not replying after 2 months.

### DISCUSSION THREAD

**Discussion by pprince9912** [community/community#126466] [Other Feature Feedback, Questions, & Ideas]
My laptop was hacked about two months ago. The hackers gained access to my LinkedIn and GitHub accounts. I regained access to LinkedIn, but GitHub suspended my account. I tried reaching out to them, but I haven't received any response, even though it's been two months. I had some projects on GitHub that I can't access now. I know this forum can't help me with my account reinstatement, but I don't know where else to go.

This is my support ticket: 2698884.

**Comment by adxthya** [comment]
Same issue! its been 2 months and my account is marked as spam. No response from github support.

**Comment by ebndev** [comment]
Hi @pprince9912,

As you mentioned, this community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
